book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+## What is cultural change?
+
+Culture is the social environment in which you partake. 
+Culture has an influence on your views, values, concerns and your identity. 
+Culture can be difficult to change as it is hard to address directly and shared values can be deeply ingrained ({cite:ps}`Kotter2012change`). 
+Instead, norms of behaviour may be easier to change. 
+
+Here, cultural change is defined as “inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation"
+
+This section will continue to describe where you will encounter cultural change. 
```

Comment:
I also wonder if there can be something added here that addresses the 'culture' of academic research environments or data science industry environments. Something that makes this connection between culture & academia/data science more explicit. 

Institutions, as collections of people that have shared norms, values, ideas, and even shared languages (i.e. technical jargon) can be defined as a 'culture'!

```suggestion
This section will continue to describe where you will encounter cultural change, especially in the context of academic research or industry.
```